typescript/infra/src/utils/utils.ts

```diff
@@ -157,3 +157,12 @@ export function writeJSON(directory: string, filename: string, obj: any) {
     JSON.stringify(obj, null, 2),
   );
 }
+
+// Returns a \ b
```

Comment:
include this in docs? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#implementing_basic_set_operations